backend/vaa-strapi/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "@strapi/typescript-utils/tsconfigs/server",
   "compilerOptions": {
+    "moduleResolution": "Node10",
```

Comment:
The 'Node' resolution has been deprecated and replaced by 'Node10'. Just a nitpick.